source/memory.tex

```diff
@@ -6156,7 +6156,7 @@
 Calling the \tcode{set_default_resource} and
 \tcode{get_default_resource} functions shall not incur a data race.
 A call to the \tcode{set_default_resource} function
-shall synchronize with subsequent calls to
+synchronizes with\iref{intro.races} subsequent calls to
```

Comment:
Please move the added  \iref to after "data race", for consistency with get_new_handler (above).